config/config.go

```diff
@@ -54,9 +55,9 @@ type Config struct {
 	Databases       map[string]string `yaml:"databases"`
 	MempoolDb       string            `yaml:"mempoolDB"`
 	BlocksDb        string            `yaml:"blocksDB"`
-	TxDb            string            `yaml:"transactionsDB"`
 	LogsDb          string            `yaml:"logsDB"`
 	MempoolSlots    int               `yaml:"mempoolSize"`
+	MemTxTimeThreshold time.Duration           `yaml:"mempoolTxTime"` //mempool tx expiration in miuntes
```

Comment:
Having this as a time.Duration feels weird. Out of the gate, a time.Duration represents nanoseconds, so we're essentially passing it around as nanoseconds until we get to the threshold test and then convert it to minutes. I would be inclined to pass it around as an int64 and convert at the last minute as `time.Duration(value) * time.Minute)` to avoid anyone who might want to use this value in the future thinking it's already a well formed duration.